src/checkbox/Checkbox.tsx

```diff
@@ -1,89 +1,90 @@
-import * as React from "react";
 import {
   CheckboxStateReturn,
   Checkbox as RenderlessCheckbox,
   CheckboxProps as RenderlessCheckboxProps,
 } from "reakit";
+import * as React from "react";
 import { cx, useControllableState } from "@renderlesskit/react";
 
 import { useTheme } from "../theme";
 import { Box, BoxProps } from "../box";
-import { createContext } from "../utils";
 import { forwardRefWithAs } from "../utils/types";
 import { CheckIcon, IndeterminateIcon } from "../icons";
 
 export type CheckboxStatus = CheckboxStateReturn["state"];
 
-export type CheckboxContextProps = {
-  defaultState?: CheckboxStatus;
+export type CheckboxStateValues = {
   state?: CheckboxStatus;
-  onStateChange?: (value: CheckboxStatus) => void;
   value?: string | number;
   size?: "xs" | "sm" | "lg";
```

Comment:
keyof Renderlesskit.getThemeValue<"checkbox", "size>

Reply:
We can't get that directly, because checkbox don't have a size object, but checkbox.icon has.

Reply:
Oh yeah right, I'll see what we can do about it. If we don't get it from the namespace then the theme extend will break

Reply:
Oh yeah you should be able to do this

```ts
keyof Renderlesskit.GetThemeValue<"checkbox", "icon">["size"];
```